File: src/main/java/com/example/healthylifestyleapp/ActivityGoalSettings.java
package com.example.healthylifestyleapp;

import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

import androidx.appcompat.app.AppCompatActivity;

public class ActivityGoalSettings extends AppCompatActivity implements View.OnClickListener {
ImageView imgBackSpace;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_goal_settings);
        imgBackSpace=findViewById(R.id.imgBackSpace);
        imgBackSpace.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        switch (v.getId())
        {
            case R.id.imgBackSpace:

        }

    }
}

File: app/src/main/java/com/example/healthylifestyleapp/ui/activities/ActivityGoalSettings.kt
package com.example.healthylifestyleapp.ui.activities

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.example.healthylifestyleapp.R
import kotlinx.android.synthetic.main.activity_goal_settings.*

class ActivityGoalSettings : AppCompatActivity(), View.OnClickListener {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_goal_settings)
        imgBackSpace.setOnClickListener(this)
    }

    override fun onClick(v: View) {
        when (v.id) {

        }

    }
}
